hr_employee_benefit/models/hr_employee_benefit_rate.py

```diff
@@ -100,22 +100,24 @@ def get_all_amount_types(self):
             ('annual', _('Annual')),
         ]
 
+    @api.multi
     def _get_amounts_now(self):
         today = context_today(self)
-        self.employee_amount = self.get_amount(today)
-        self.employer_amount = self.get_amount(today, employer=True)
-
+        for rate in self:
+            rate.employee_amount = rate.get_amount(today)
+            rate.employer_amount = rate.get_amount(today, employer=True)
+    
     @api.multi
     def get_amount(self, date, employer=False):
-        self.ensure_one()
```

Comment:
IMHO a `api.multi` with a return not is the best solution.
You could use `ensure_one` from `get_amount` method because with your change now is used with a singleton from `_get_amounts_now`